Sources/BraintreePayPalNativeCheckout/BTPayPalNativeTokenizationClient.swift

```diff
@@ -33,18 +33,19 @@ class BTPayPalNativeTokenizationClient {
         ) { body, _, error in
             guard let json = body, error == nil else {
                 let underlyingError = error ?? BTPayPalNativeError.invalidJSONResponse
-                self.apiClient.sendAnalyticsEvent("ios.paypal-native.tokenize.tokenize-url-request.failed")
+                self.apiClient.sendAnalyticsEvent(BTPayPalNativeCheckoutAnalytics.tokenizeUrlRequestFailed)
```

Comment:
I think for this we can send the general `tokenizeFailed`

Reply:
This is where we send a specific tokenize fail. The general fail is handled in` BTPayPalNativeCheckoutClient` line 171, I think,  on error return from this` BTPayPalNativeTokenizationClient`'s tokenize function.

Reply:
My understanding was that `tokenize:failed` will be general fail event and that alone is the total failure of the entire flow ending with a token.